src/routes/login/index.js

```diff
@@ -1,24 +1,59 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
 import { withRouter } from "react-router";
-import { Content, Heading, Paragraph, Section } from "styled";
+import { getAuthUrl } from "adapters";
+import {
+  Button,
+  Content,
+  CoverImage,
+  Container,
+  Heading,
+  Header,
+  Paragraph,
+  Section
+} from "styled";
+import Typed from "components/Typed";
+
+import screenLG from "media/dashboard.png";
+import screenMD from "media/dashboard-md.png";
+import screenSM from "media/dashboard-sm.png";
 
 class Login extends Component {
   componentDidMount() {
     if (this.props.auth.toJS().isAuthenticated) {
       this.props.router.push("/dashboard");
     }
   }
+
   render() {
     return (
-      <Section>
-      <Content>
-        <Heading>Start your first project</Heading>
-        <Paragraph>
-          Sign up using Dropbox and start using Anni to track your project's progress!
-        </Paragraph>
-      </Content>
-    </Section>
+      <Section split>
+        <Header>
+          <Button icon="logo" to="/" noBorder noHover />
+        </Header>
+        <Container split={60} center>
+          <Content>
+            <Heading mb={16}>
+              Finally, a single space for
+              {" "}
+              <Typed data="hello" />
+              {" "}
+              to manage and present their entire workflow.
+            </Heading>
+            <Paragraph responsive strong>
+              Write a script, create a moodboard, annotate your storyboards, showcase your styleframes, present your video and collaborate with your client in a single space.
+              <br />
+              <br />
+              <Button href={getAuthUrl()}>Sign In</Button>
+            </Paragraph>
+          </Content>
+        </Container>
+        <Container split={40} center media>
+          <CoverImage src={screenLG} alt="dashboard" desktop />
```

Comment:
Hmm.. I'm not sure about this approach. A user must download all 3 image assets regardless of which viewport size their device is. We should use the picture element:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

It's not supported by IE, but IE is deprecated in favor of Edge anyways: https://caniuse.com/#feat=picture